proto/rill/ui/v1/dashboard.proto

```diff
@@ -72,62 +72,58 @@ message DashboardState {
   // Dimension filters applied
   rill.runtime.v1.MetricsViewFilter filters = 2;
   // Expression format for dimension filters
-  rill.runtime.v1.Expression where = 20;
+  rill.runtime.v1.Expression where = 3;
```

Comment:
Changing the index will break backwards compatibility. Any specific reason to do this?